packages/mui-material-next/src/styles/Theme.types.ts

```diff
@@ -133,6 +133,45 @@ export interface Shapes {
   borderRadius: number;
 }
 
+export interface MD3Easing {
+  linear: string;
+  standard: string;
+  standardAccelerate: string;
+  standardDecelerate: string;
+  emphasized: string;
+  emphasizedDecelerate: string;
+  emphasizedAccelerate: string;
+  legacy: string;
+  legacyDecelerate: string;
+  legacyAccelerate: string;
+}
+export interface MD3Duration {
+  short: number[];
```

Comment:
Alternatively we can go with this:

```suggestion
  short1: number;
  short2: number;
  short3: number;
  short4: number;
```
Same for the others keys.

Reply:
Wouldn't a tuple be more accurate in this case (`[number, number, number, number]`)?

Reply:
Honestly, I haven't used a tuple because that showed in md2 to be problematic, with the shadows, as when extending the design system, you may want to limit or extend the list of items specified. Having a module augmentation just for this seems like overkill. What do you think?

Reply:
`short1` - `short4` will also suffer from this, no? If so, let's stick to an array.

Reply:
I think it should be:

```ts
{
  short1: string;
  ...
  short4: string;
}
```

In theme, it looks like this:
```js
{
  short1: '50ms', // valid CSS unit
}

transition: 'ease-in var(--md.sys.motion.duration.short1)',
```

Reply:
Ok, let's follow strictly the MD spec.

Reply:
Updated, I've also added the support to generate vars for everything in `motion` except `create` and `getAutoHeightDuration`. Looks better now.